.yarnclean

```diff
@@ -0,0 +1,3 @@
+# Working around styled-components types issue as suggested in
+# https://github.com/DefinitelyTyped/DefinitelyTyped/issues/33311/#issuecomment-546727329
+@types/react-native
```

Comment:
@ShimiSun Not sure if there were concerns about this workaround or not, but one alternative to this could be to set the following in tsconfig.json:

```
"types": [],
"lib": ["dom", "es6", "es2016"],
```

My understanding is that this way TypeScript will not implicitly load all type definitions, but only ones listed in "types".

* Set types to  `[]` so no types are loaded implicitly
* When a type is needed we `import` it instead (this is what is already being done in this project, as well as what I do in my own projects)
* Add in the ECMAScript and DOM APIs that we support in "lib" since they are no longer loaded implicitly

Reply:
Thinking about this a little more, I do like the solution of setting types to `[]` a little better than relying on yarn to remove the @types/react-native package after install.

If there are no concerns with that I can update this PR with that change.